src/main/java/org/opensearch/knn/index/engine/KNNMethodContext.java

```diff
@@ -20,7 +14,7 @@
 import org.opensearch.core.common.io.stream.StreamInput;
 import org.opensearch.core.common.io.stream.StreamOutput;
 import org.opensearch.core.common.io.stream.Writeable;
-import org.opensearch.knn.index.util.KNNEngine;
+import org.opensearch.knn.index.SpaceType;
```

Comment:
Again, not engine specific

Reply:
Lets discuss on https://github.com/opensearch-project/k-NN/pull/1913#discussion_r1698810135